src/CTA.WebForms2Blazor/ControlConverters/GridViewControlConverter.cs

```diff
@@ -21,7 +72,32 @@ public GridViewControlConverter() : base()
 
         public override HtmlNode Convert2Blazor(HtmlNode node)
         {
-            throw new NotImplementedException();
+            node.OwnerDocument.OptionOutputOriginalCase = true;
+            
+            var itemTypeAttr = node.Attributes.AttributesWithName("itemtype").FirstOrDefault();
+            bool hasItemType = itemTypeAttr != null;
+            var itemType  = itemTypeAttr?.Value ?? "PleaseReplaceWithItemTypeHere";
+
+            IEnumerable<Attribute> newItemTypeAttr = new List<Attribute>()
+            {
+                new Attribute("ItemType", itemType)
+                //"ItemType=" + itemType
+            };
+            foreach (string subControl in SubControls)
+            {
+                //Todo: make sure not adding itemtype if already exists
+                UpdateInnerHtmlNode(node, "asp:" + subControl, newName: subControl, newAttributes: newItemTypeAttr);
```

Comment:
A lot of string literals in this method, may want to make some/all of them into constants as some of them like "ItemType" seem to be reused a lot. Also, the "asp:" prefix seems like a constant you might want to put in the Constants.cs file

Reply:
I agree, I only had it that way for ease of development but I'll refactor all that stuff soon